### ISSUE TITLE
Image Caption/Alternative text/Title text (tooltip) not showing on page save/publish

### ISSUE THREAD

**Issue by amolswnz** [silverstripe/silverstripe-elemental#920]
## Steps to reproduce
- Create a new content block 
- Add image
- Add Caption / Alternative text / Title text (tooltip) 
- Save/Publish
- Click on Edit image
- Caption/Alternative text/Title text (tooltip) all do not show up

## Tested on following version
```
dnadesign/silverstripe-elemental 4.6.0
silverstripe/asset-admin         1.8.0
silverstripe/assets              1.8.0
silverstripe/cms                 4.8.0
silverstripe/framework           4.8.0
```

**Comment by emteknetnz** [comment]
Cannot replicate, alt text is still showing for me in the html source in tinymce and the edit image panel

`<p><img class="leftAlone ss-htmleditorfield-file image" src="/assets/sample-tui.jpg" alt="my alt text" width="600" height="480" data-id="1" data-shortcode="image" /></p>`

**Comment by brynwhyman** [comment]
> Cannot replicate

+1, just tried this with recipe-cms 4.8.0 and recipe-cms 4.8.x and can't replicate the bug.

**Comment by brynwhyman** [comment]
@amolswnz I'll close this for now. Please keep the conversation going if you believe this is still an issue and I'll reopen it.

**Comment by amolswnz** [comment]
@brynwhyman https://ibb.co/RTWRw91

**Comment by emteknetnz** [comment]
Thanks for the video @amolswnz 

Summary of video:
- Alt text HTML in view source is getting updated
- When opening the Edit Image panel for a second time, alt text field is not being filled in

Looks like the issue is something to do with this again - https://github.com/silverstripe/silverstripe-asset-admin/pull/1213/files

**Comment by emteknetnz** [comment]
Hey @amolswnz - I've looked into this issue - it was fixed in silverstripe/asset-admin [1.8.1](https://github.com/silverstripe/silverstripe-asset-admin/releases/tag/1.8.1) with [this commit](https://github.com/silverstripe/silverstripe-asset-admin/commit/91d594575ad279a2b0610d7b12a5d3d468fe5cdd)

I recommend that you update silverstripe/asset-admin to the latest 1.8.x version (currently 1.8.3)

If you are using silverstirpe/recipe-cms 4.8.0, since that's fixed to silverstripe/asset-admin 1.8.0, you'll need to use an alias
`"silverstripe/asset-admin": "1.8.3 as 1.8.0"`

I'll close this issue now. Llet me know if you have any further issues
